releasenotes/notes/0.14/add-title-to-pulse-drawer-69db1ddb2f696367.yaml

```diff
@@ -7,7 +7,7 @@ feature:
     :class:`~qiskit.pulse.Schedule` instance's ``name`` property.
     The fontsize is controllable with the pulse drawer style sheet. For example:
 
-    .. jupyter-execute::
+    .. parsed-literal::
```

Comment:
I would probably use `.. code-block:: python` or delete this line and change to `For example::` so that there is still syntrax highlighting. parsed-literal is just monospace font with exact spacing.

Reply:
thank you!